tox.ini

```diff
@@ -24,7 +24,36 @@ commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/nvtabular.git@{posargs:main}
     python -m pytest --cov-report term --cov merlin -rxs tests/unit/
 
+[testenv:py38-multi-gpu]
+; Runs in: Github Actions
+; Runs multi-GPU tests.
+requires = tox-conda
+passenv =
+    CUDA_HOME
+setenv =
+    TF_GPU_ALLOCATOR=cuda_malloc_async
```

Comment:
Given we only use horovod for dataloader we should be able to set this information in our environment. We could put this information in documentation instead of running this in our testing.